UIMentoringSession/MenteesChallenges/Pets Main Card UI Chellenge/OleksandrKharchenko/AnimalsCells/Olkha_FilterAnimalsCell.xib

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
No width and height for imageView. No trailing for the label. No top/bottom constraints for the whole cell 
![Screenshot 2023-04-17 at 14 13 25](https://user-images.githubusercontent.com/14887101/232468654-3c6b33c8-035f-4dd9-b434-38e8d09adc22.png)